<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Return</class>
 <widget class="QDialog" name="Return">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>还书界面</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>想要还的书ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="OK_Button">
       <property name="text">
        <string>还书</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="info">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yearEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="monthEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="dayEdit"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>78</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_Button">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Cancel_Button</sender>
   <signal>clicked()</signal>
   <receiver>Return</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
